config/worker.js

```diff
@@ -0,0 +1,146 @@
+const CLIENT_ID = '6caaf5d0a58e47868553160955e5b8e4'
+const CLIENT_SECRET = 'REDACTED'
+const REDIRECT_URL = '/spotifycb'
+const WORKER_URL = 'https://duoco.cawman.workers.dev'
```

Comment:
That's something that I'm definitely not gonna allow to be in this code. This project shouldn't depend on a third-party website, where I don't even know the code behind it.

Reply:
The website is run by https://workers.cloudflare.com/.

Reply:
And i cannot change the websites code even if i wanted to.

Reply:
Do we really need this "workers" stuff or could we use it without?

Reply:
Don't mark it as resolved if it isn't resolved.. also please answer my question.